### ISSUE TITLE
[feature request] Trend for Team effectiveness measure from different retrospectives 

### ISSUE THREAD

**Issue by fralon** [microsoft/vsts-extension-retrospectives#314]
I am missing a trending chart to track Team Effective Measure in AzDO Retrospectives. This could potentially be showed in the History tab to measure how Team effectiveness is developing over time. You currently need to collect manually the results for each retro, and potentially put it in a different program to create a chart...

**Comment by AHGambiSno** [comment]
Agreed,  For this to be useful for monitoring scores over time, there needs to be a better mechanism than copying and pasting the info out of the summary and into another tool.  
It would also be good to know how many of the team participated in this scoring in each retrospective.

**Comment by dstoddar** [comment]
Thank you for your feedback. We are in the process of implementing additional reporting capabilities and trending is under evaluation!

**Comment by ArthurManz** [comment]
Any feedback from the team assessment on this one?

**Comment by dphanson63** [comment]
Team assessment history exists in basic form on the History tab.  Future releases may elaborate on this report.
